### DISCUSSION TITLE
Complete list of presentWeather + textDescription field values

### DISCUSSION THREAD

**Discussion by kfeinberg** [weather-gov/api#214] [Q&A]
Hello! Is there a complete list of all possible values that the textDescription and presentWeather fields could have? (ex: https://api.weather.gov/stations/KIND/observations)

**Comment by hluker** [comment]
@kfeinberg,
For present weather, it's derived from the METAR code for present weather -- here's an example. https://weather.cod.edu/notes/metar.html#wx. 

For text description, I believe they match up with what's found here https://www.weather.gov/forecast-icons
